requirements_full.txt

```diff
@@ -0,0 +1,17 @@
+# Optional requirements - used by various plugins or additional features like mappings
+#
+# Note: Some important indirect dependencies are not listed here.
```

Comment:
The idea with this note was to then list _what_ indirect dependencies were important that are not listed here - the only one I can see now not listed would be `jsonschema`, which could then be listed below this note (commented out).

Reply:
My point here being. There is no reason to have this note "teaser" if no indirect dependencies are then not listed. So I'd suggest either removing this note again or add some of those mentioned "important indirect dependencies"